wire-ios/Wire-iOS/Sources/Authentication/Coordinator/AuthenticationCoordinator.swift

```diff
@@ -850,10 +850,10 @@ extension AuthenticationCoordinator {
 
         guard let session = statusProvider.sharedUserSession else { return }
         let e2eiCertificateUseCase = session.enrollE2EICertificate
-        guard let rootViewController = AppDelegate.shared.window?.rootViewController else {
+        guard let topmostViewController = UIApplication.shared.topmostViewController() else {
```

Comment:
Maybe it's okay to use `topmostViewController(onlyFullScreen:)` as a quick-fix now, but we should get rid of this soon.

Have you checked the implementation about the Bool argument? Shouldn't it be set to `false`?
The check `modalPresentationStyle == .fullScreen` doesn't look good, in my opinion.

Reply:
I would probably take the `targetViewController` as argument in the `func startE2EIdentityEnrollment`, but feel free to keep it like that for now.